mount/index.ts

```diff
@@ -84,18 +100,16 @@ function doMount (elements: HTMLElement[], mountable: mojave.Mountable, rawOptio
             }
             else if ("class" === options.type)
             {
-                let standaloneOptions = options as mojave.ClassMountOptions;
-                const mounted = new mountableAny(node, ...(standaloneOptions.params || []));
+                let mountableClass = mountable as mojave.MountableClass;
+                let standaloneOptions = options as mojave.ClassMountOptions<any>;
+                const mounted = new mountableClass(node, ...(standaloneOptions.params || []));
```

Comment:
Do we really need this change? Isn't it enough to change the type declaration at the "interface functions" (= exported functions)

Reply:
Ah, you just moved the variable into this `if`. Nevermind then